sys/vm/vm_cheri_revoke.c

```diff
@@ -83,6 +83,16 @@ SYSCTL_COUNTER_U64(_vm_stats_cheri_revoke, OID_AUTO, restarts, CTLFLAG_RD,
     &cheri_revoke_restarts,
     "Number of VM map re-lookups");
 
+static COUNTER_U64_DEFINE_EARLY(cheri_skip_prot_no_readcap);
+SYSCTL_COUNTER_U64(_vm_stats_cheri_revoke, OID_AUTO, skip_prot_no_readcap, CTLFLAG_RD,
+    &cheri_skip_prot_no_readcap,
+    "Virtual pages skipped in map entries without readcap permission");
+
+static COUNTER_U64_DEFINE_EARLY(cheri_skip_obj_no_hascap);
+SYSCTL_COUNTER_U64(_vm_stats_cheri_revoke, OID_AUTO, skip_obj_no_hascap, CTLFLAG_RD,
+    &cheri_skip_obj_no_hascap,
+    "Virtual pages skipped in VM objects with no capabilities");
+
```

Comment:
Reason to prefer this over the caprevoke stats infrastructure?  (Though admittedly I'd not be sad to see the latter get ripped out.)

Reply:
Mostly that I wanted to have a global view of the counter value, and sysctl is more convenient for that. The existing stats infrastructure collects per-process stats and I believe is limited to printing them when the process exits. The right direction might be a hybrid scheme wherein we maintain per-process (really, per-vmspace) and global counters, and use the former to update the latter after each scan.